libminifi/include/core/logging/Logger.h

```diff
@@ -234,33 +190,34 @@ class Logger : public BaseLogger {
   std::mutex mutex_;
 
  private:
+  std::string trimToMaxSizeAndAddId(std::string&& my_string) {
```

Comment:
```suggestion
  std::string trimToMaxSizeAndAddId(std::string my_string) {
```

Reply:
Fixed in https://github.com/apache/nifi-minifi-cpp/pull/1670/commits/fea09110f9e9c60abee47351c52a1cfec2d87d8a